src/LightningDB/LightningDB.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>LightningDB</Description>
     <VersionPrefix>0.14.0</VersionPrefix>
     <Authors>Ilya Lukyanov;Corey Kaylor</Authors>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>

```

Comment:
There is no reason to add a target. For instance netstandard2.0 covers net6.0.
Unless ... you are updating dependency, or a direct dependency targets it too.

Note: Here you can remove the dependency on System.Memory for netcoreapp31 and greater.